test/extended/util/proxy.go

```diff
@@ -0,0 +1,19 @@
+package util
+
+import (
+	"context"
+
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// HasProxy returns whether or not the currently installed cluster has a proxy configured
+func HasProxy(oc *CLI) (bool, error) {
```

Comment:
Perhaps need to make it clearer that we're talking about the cluster?

```suggestion
// IsClusterProxyEnabled returns true if the cluster has a global proxy enabled
func IsClusterProxyEnabled(oc *CLI) (bool, error) {
```

Reply:
updated in next push